### PR TITLE
Update Pod::Checker to CPAN version 1.74

### PR DESCRIPTION
**Pull request by toddr**
[DELTA]

+ CPAN#133312: Add strict compliance to Pod-Checker
  Thanks TODDR for the patch to make the code "use strict" compliant
+ CPAN#98326: Can we make “A non-empty Z<>” a warning and not an error
  This is a corner case... and since Pod::Simple anyway "nixes" anything
  inside Z<...>, this is a warning, not an error. Mind the caveats.
+ Added an additional test to achieve 100% statement coverage and added
  some POD to also have 100% POD coverage
+ Made all text files unixish, no CRLF

### CODE REVIEW

File: cpan/Pod-Checker/t/pod/testpchk.pl [hunk 1]
```diff
@@ -14,7 +16,7 @@ BEGIN
 }
 
 use Pod::Checker;
-use vars qw(@ISA @EXPORT $MYPKG);
+use vars qw(@ISA @EXPORT @EXPORT_OK $MYPKG);
```
Comment (atoomic) [inline cpan/Pod-Checker/t/pod/testpchk.pl:19]: maybe time to drop `vars` :-)

**Review by atoomic — APPROVED** [review]

File: cpan/Pod-Checker/t/pod/testpchk.pl (hunk 1 above)
Comment (toddr) [reply cpan/Pod-Checker/t/pod/testpchk.pl:19]: That would be an upstream request.
